### PR TITLE
Adjust AgGrid results for LF and AS

### PR DESCRIPTION
**Pull request by souissimai**

### CODE REVIEW

**Review by trikimahdi — APPROVED** [review]
tests ok
code ok

File: src/components/security-analysis-result.js [hunk 1]
```diff
@@ -412,57 +419,57 @@ const SecurityAnalysisResult = ({ onClickNmKConstraint, result }) => {
     const nmKConstraintsColumns = useMemo(() => {
         return [
             {
-                width: 200,
+                flex: 1,
```
Comment (ghelloumi) [inline src/components/security-analysis-result.js:422]: you can add flex: 1 only on columnDefs
for both renderTableNmKConstraints and renderTableNmKContingencies (then, you can remove width for columnsNmKContingencies)

**Review by archendel — CHANGES_REQUESTED** [review]
Load sorting is not sorting correctly
It should sort data as numbers but sorts them alphabetically

**Review by ghelloumi — APPROVED** [review]
tests ok
code ok

**Review by archendel — APPROVED** [review]

**Pull request merged**